service/src/main/java/uk/gov/hmcts/reform/fpl/service/PlacementService.java

```diff
@@ -81,4 +85,23 @@ private Placement removeConfidentialDocuments(Element<Placement> placement) {
             .filter(placement -> placement.getValue().getChildId().equals(childId))
             .findFirst();
     }
+
+    public List<String> getBinaryUrlsForOrderAndNotices(List<Element<Placement>> placements,
+                                                        PlacementOrderAndNotices.PlacementOrderAndNoticesType type) {
```

Comment:
Why not PlacementOrderAndNoticesType` static import?

Reply:
I don't think I can do this here? @Jamesnelson117

Reply:
It can